### DISCUSSION TITLE
Unable to login to Airbyte Web Interface after installation

### DISCUSSION THREAD

**Discussion by gauravshah01** [airbytehq/airbyte#44043] [Deployment]
Hi,

We've installed Airbyte on a Linux EC2 instance using abctl, opened port 8000. Also updated the server FQDN for Nginx Ingress hostname and retrieved credentials using "abctl local credentials". However when I open the url in browser(chrome) http://HOSTNAME:8000 and enter the email and password, I get a "Invalid username or password" error. Saw a similar issue reported below, but could not update it as it's a read-only knowledge article:

https://discuss.airbyte.io/t/trouble-logging-in-after-installing-airbyte-locally/7511

abctl version: v0.11.1

Kindly help and let me know if additional details are needed.

**Comment by marcosmarxm** [comment]
Can you check the password using `abctl local credentials` and the email from the `organization` table?
Please check more in [this](https://github.com/airbytehq/airbyte/discussions/44021) discussion. I'm closing this as it is very similar topic.

Please add comments and continue the discussion there.
